src/components/ScriptureWorkspaceCard.js

```diff
@@ -41,7 +41,8 @@ export default function ScriptureWorkspaceCard({
   const onRefSelectClick = ({sourceId, bookId: bookIdFromEditor, chapter, verse}) => {
     const normalizedBookId = (bookIdFromEditor || bookId).toLowerCase()
     setBibleReference({ sourceId, bookId, chapter, verse })
-    bRefActions.goToBookChapterVerse(bookId.toLowerCase(), chapter.toString(), verse.toString())
+    bRefActions.applyBooksFilter([bookId?.toLowerCase()])
+    bRefActions.goToBookChapterVerse(bookId?.toLowerCase(), chapter?.toString(), verse?.toString())
```

Comment:
`bookId?.toLowerCase()` is being used in multiple places. Do you think it would be more readable to have a `const` declaration for this expression instead?

Reply:
Yes, actually normalizedBookId will probably work here. I changed it now.